### PR TITLE
Add app required models

### PR DESCRIPTION
**Pull request by cedricbahirwe**

### CODE REVIEW

**Review by esthcarelle — COMMENTED** [review]
Why having a companion object inside a data class model?

**Comment by cedricbahirwe** [comment]
> Why having a companion object inside a data class model?

Please, Which File exactly?

**Review by esthcarelle — CHANGES_REQUESTED** [review]
Can we have the strings not hardcoded sil te plait?

File: TheDialer/app/src/main/java/com/cedricbahirwe/dialer/model/Contact.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+data class Contact(val names: String, private var phoneNumbers: List<String>) {
+
+    private val id: String
+    get() { return names + phoneNumbers.toString() }
+
+    fun updatePhones(numbers: List<String>) {
+        phoneNumbers = numbers
+    }
+    companion object {
```
Comment (esthcarelle) [inline TheDialer/app/src/main/java/com/cedricbahirwe/dialer/model/Contact.kt:11]: why having a companion object inside a data class model?

**Comment by cedricbahirwe** [comment]
> Can we have the strings not hardcoded sil te plait?

Please mention the file, or add a comment from the file directly so I can see it

**Review by esthcarelle — CHANGES_REQUESTED** [review]

File: TheDialer/app/src/main/java/com/cedricbahirwe/dialer/model/Contact.kt (hunk 1 above)
Comment (cedricbahirwe) [reply TheDialer/app/src/main/java/com/cedricbahirwe/dialer/model/Contact.kt:11]: I have replied in the above comment, Does it make sense or Am I Missing something?

[the remark this answers is not in the document]
Comment (esthcarelle) [reply TheDialer/app/src/main/java/com/cedricbahirwe/dialer/model/Contact.kt:11]: ooh okay then I think it s better when it is a separate class that has all examples

Comment (cedricbahirwe) [reply TheDialer/app/src/main/java/com/cedricbahirwe/dialer/model/Contact.kt]: I have resolved this

[the remark this answers is not in the document]
Comment (cedricbahirwe) [reply TheDialer/app/src/main/java/com/cedricbahirwe/dialer/model/Contact.kt]: I have resolved this

**Review by esthcarelle — APPROVED** [review]

**Pull request merged**